molgenis-data/src/main/java/org/molgenis/data/support/DynamicEntity.java

```diff
@@ -199,25 +198,24 @@ public void set(Entity values)
 	}
 
 	/**
-	 * Validate is value is of the type defined by the attribute data type.
+	 * Validate if value is of the type defined by the attribute data type.
 	 *
 	 * @param attrName attribute name
 	 * @param value    value (must be of the type defined by the attribute data type.)
 	 */
 	protected void validateValueType(String attrName, Object value)
 	{
-		if (value == null)
-		{
-			return;
-		}
+		if (value == null) return;
 
 		AttributeMetaData attr = entityMeta.getAttribute(attrName);
 		if (attr == null)
 		{
-			throw new UnknownAttributeException(format("Unknown attribute [%s]", attrName));
+			// This attribute does not exist yet, this can be because of a i18n situation
```

Comment:
The validation is meant to check for cases where you set data for non-existing attributes. Consider fixing this problem upstream such that setting data for non-existing attributes does not occur.